playbooks/web.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Should be call the file something like `install_palava_web.yml` to be consistent with `install_signaltower`

Reply:
right, good point, although let's maybe remove the install_ prefix, so it is:

- signaltower.yml
- palava_web.yml

What do you think about this? @erikzenker @juwi

Reply:
I dont really care. Important for me is the consistency and that the usage is documented.

Reply:
The `install` prefix suggest that the playbook can only install. If we drop the prefix then it should also be capable of updating etc. (general role)

Reply:
I will just add the `install` prefix